### PR TITLE
Add a loading screen for browser and desktop IDE

### LINKED ISSUE #688: As a user I want a loading screen so that I understand that IDE is starting
### Rationale

XOD IDE is heavy and may take up to dozen seconds (for the first visit) to load and start. Currently, all a user sees is a blank white screen. It distracts UX.

We should make some starting screen that shows up immediately so that all other preparations would be done behind it.

### Acceptance criteria

- [x] Once I launch browser IDE with browser cache cleared, I see full width and height Loading… screen
- [x] When loading is done the screen switches to the IDE interface instantly

### Out of scope

- Progress bar
- Desktop IDE screen

Although, they would be nice if could be implemented trivially.

### PR DESCRIPTION
**Pull request by evgenykochetkov**
Closes #688

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "bootstrap": "lerna bootstrap",
     "build": "node ./tools/lerna-run-sorted build",
     "clean": "yarn run clean:dist && yarn run clean:node_modules",
-    "clean:dist": "rimraf packages/*/dist",
+    "clean:dist": "rimraf packages/*/dist packages/xod-cli/bin packages/xod-client-electron/src-babel",
```
Comment (nkrkv) [inline package.json]: Oops :+1:

File: packages/xod-client-browser/webpack.config.js [hunk 2]
```diff
@@ -89,7 +89,7 @@ module.exports = {
   plugins: [
     new webpack.NoErrorsPlugin(),
     new CopyWebpackPlugin([
-      { from: pkgpath('src/index.html') },
+      { from: pkgpath('../xod-client/src/core/assets/index.html') },
```
Comment (nkrkv) [inline packages/xod-client-browser/webpack.config.js]: The `..` could lead to bad things if we’ll reorganize packages some day. Better use:

```javascript
pkgpath('node_modules/xod-client/src/core/assets/index.html')
```

File: packages/xod-client-electron/webpack.config.js [hunk 3]
```diff
@@ -103,7 +103,7 @@ const options = {
   plugins: [
     new webpack.NoErrorsPlugin(),
     new CopyWebpackPlugin([
-      { from: pkgpath('src/index.html') },
+      { from: pkgpath('../xod-client/src/core/assets/index.html') },
```
Comment (nkrkv) [inline packages/xod-client-electron/webpack.config.js]: Ditto

File: packages/xod-client-browser/webpack.config.js (hunk 2 above)
Comment (evgenykochetkov) [inline packages/xod-client-browser/webpack.config.js]: done 👍🏻

**Pull request merged**